src/main/groovy/life/qbic/portal/sampletracking/app/samples/query/SampleTrackingQueryDataSource.groovy

```diff
@@ -1,11 +1,14 @@
 package life.qbic.portal.sampletracking.app.samples.query
 
 import life.qbic.datamodel.samples.Location
+import life.qbic.datamodel.samples.Sample
 
 interface SampleTrackingQueryDataSource {
 
     Location currentSampleLocation(String sampleId)
 
     List<Location> availableLocationsForPerson(String emailAddress)
 
+    Sample queriedSample(String sampleId)
```

Comment:
we could maybe replace this by calls to `currentSampleLocation` and associating this with the sampleId in the presenter (which we have to add since our view model does the job right now).